### ISSUE TITLE
Jukeboxes don't play any music

### ISSUE THREAD

**Issue by mhsjlw** [cuberite/cuberite#3370]
Client version: 1.10.(1,2), 
Server OS: Linux
Commit id: f7029eddd4120f672c59a4b3a4395832f4de8863

When placing a record into a Jukebox
### Expected behavior

Takes disk and plays the audio while showing the user the title bar of the song that is currently playing
### Actual behavior

Takes disk, doesn't play an audio
### Steps to reproduce the behavior

Place a Jukebox, put a disk inside of it

In 1.9, `0x46` is sign update, but in 1.10 it is a sound packet. `0x19` is the named sound packet. `0x46` for 1.10 needs to be implemented then, check if the client is 1.8 or 1.9 and function as it currently does, but 1.10 needs to have a sound packet sent to the client after updating the block's meta and and the sound effect. Note that 

``` cpp
m_World->BroadcastSoundEffect("record_cat", m_PosX, m_PosY, m_PosZ, 0.2, 0.5);
```

doesn't actually play the "record_cat" disk as you'd think it would

**Comment by Pokechu22** [comment]
Jukeboxes don't use the play sound packet; wrong IDs for that aren't the cause of the issue.  Instead, they use the [effect packet](http://wiki.vg/Protocol#Effect) (`BroadcastSoundParticleEffect`), which has hardcoded IDs.  Unfortunately cuberite's [enum for those IDs](https://github.com/cuberite/cuberite/blob/f7029eddd4120f672c59a4b3a4395832f4de8863/src/EffectID.h) has hardcoded values, and those values are no longer correct as of 1.9 - for instance, play record is 1010, not 1005.  Since the wrong IDs are used, nothing with the effect packet will actually work correctly (eg, during 1.9 development hard fall particles would play the end crystal explosion sound, for instance, although that's been disabled).

> Note that 
> ```cpp
> m_World->BroadcastSoundEffect("record_cat", m_PosX, m_PosY, m_PosZ, 0.2, 0.5);
> ```
> doesn't actually play the "record_cat" disk as you'd think it would

That's because BroadcastSoundEffect expects sound effect IDs, not item IDs &mdash; use `record.11`, `record.13`, `record.blocks`, `record.cat`, `record.chirp`, `record.far` `record.mall`, `record.mellohi`, `record.stal`, `record.strad`, `record.wait`, and `record.ward`.  However, playing records this way doesn't display the flashing "Now playing" text.  (A list of sound IDs can be found in vanilla's sounds.json or via [burger](http://pokechu22.github.io/Burger/1.10.html#sounds)).

**Comment by LogicParrot** [comment]
If anyone is dissatisfied with the current multiprotocol handling, discuss here: https://github.com/cuberite/cuberite/issues/3201

**Comment by peterbell10** [comment]
Fixed by #3422.
